src/test/centos/Dockerfile.in

```diff
@@ -23,9 +23,7 @@ FROM centos:%%os_version%%
 COPY install-deps.sh ceph.spec.in /root/
 # http://jperrin.github.io/centos/2014/09/25/centos-docker-and-systemd/
 RUN yum -y swap -- remove fakesystemd -- install systemd systemd-libs && (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done) && rm -f /lib/systemd/system/multi-user.target.wants/* && rm -f /etc/systemd/system/*.wants/* && rm -f /lib/systemd/system/local-fs.target.wants/* && rm -f /lib/systemd/system/sockets.target.wants/*udev* && rm -f /lib/systemd/system/sockets.target.wants/*initctl* && rm -f /lib/systemd/system/basic.target.wants/* && rm -f /lib/systemd/system/anaconda.target.wants/* && yum install -y redhat-lsb-core
-RUN yum install -y wget
-RUN wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
-RUN rpm -ivh epel-release-7-5.noarch.rpm
+RUN yum install -y yum-utils && yum-config-manager --add-repo https://dl.fedoraproject.org/pub/epel/7/x86_64/ && yum install --nogpgcheck -y epel-release && rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
```

Comment:
It's not strictly necessary to clean up the file. In fact, you'll have EPEL activated in the container as soon as you run `yum-repo-manager`, so in that sense, it's not *strictly* necessary even to install `epel-release`. (You'd lack the GPG key, but at least yum could still operate and pull packages down at a bare minimum.)

The purpose of deleting the temporary `.repo` file is to make sure we're using the latest EPEL repository configuration as much as possible. Sometimes the `epel-release` maintainers will ship some small improvements to their `epel.repo` file, for example. By deleting our temporary `.repo` file, we will make 100% sure that we're using the latest EPEL repository configuration in order to take advantage of the latest updates. When there's two `.repo` files that do the same thing, it's less clear what will happen.

This is really a very small corner case, but I do think the clean up here is going to avoid any potential unforseen problems that would arise from having two almost-the-same-but-not-quite `.repo` files.